src/shared/components/layout/StandardAppLayout.scss

```diff
@@ -130,6 +130,11 @@ $drawerWindowWidth: 45px;
   margin-bottom: 30px;
 }
 
+.navigateIcon {
+  width: 26px;
+  height: 26px;
+}
```

Comment:
Wait, you still had all this padding (green) to shave off before starting tweaking width and height.

![Screenshot 2022-11-04 at 15 12 57](https://user-images.githubusercontent.com/6834224/200010227-8953e1e9-18ad-4450-8d86-915e93d4b9c8.png)

BTW: might be worth waiting until the updated ImageButton component is merged; and check how it works there.

Reply:
So the padding is set to 4px on all the icons....whats the better solution, set the padding to 2px or set the height and width? Feels like we might want to keep the padding for the icons the same isn't it? I

Reply:
The updated `ImageButton` component has `--image-button-svg-width` and `--image-button-svg-height` properties. You can use either them or the padding to tweak the icon size; whichever is more convenient/appropriate.

Reply:
I went with the padding instead of readjusting the width and height because the padding was affecting the size of the image so that sounds like a better fix.

Reply:
> so that sounds like a better fix

I'm confused by why you would consider this a better fix; and drew a diagram to explain what I meant:

![image](https://user-images.githubusercontent.com/6834224/200355470-1d63d9e9-cccd-4685-a11f-dc93f6a0c676.png)

I'm not saying that this is better or worse than the padding; I'm just saying that this allows you to control the dimensions of the svg directly, rather than calculating them from image button width(height) - padding *2.

Reply:
I am also confused. I understand your point of being able to override the image width and height using the css variable. So which one should we go with:
* set padding to 2px or
* set width and height to 24px

Reply:
24x24px is the dimensions of the ImageButton, which is this rectangle:

<img src="https://user-images.githubusercontent.com/6834224/200360473-7911e217-625e-47b0-a31d-bba629682abc.png" width="400px" />

whereas the `--image-button-svg-width` and `--image-button-svg-height` properties control the size of the svg image inside the rectangle:

<img src="https://user-images.githubusercontent.com/6834224/200360886-33ae4140-a1c9-4c1d-976b-cbf7559bf9c4.png" width="550px" />

Both the padding and the CSS variable are valid options for setting the size of the svg image inside the button. The option with the variable is more straightforward because it is easier to reason about than the padding option (e.g. `--image-button-svg-width: 20px` and `--image-button-svg-height: 20px` gives you an svg with a size of 20x20px; whereas `padding: 4px` means that the size of the svg is the size of the ImageButton (24x24px) minus twice the size of the padding (-4px left, -4px right, -4px top, -4px bottom)).